hpo_demo_setup.sh

```diff
@@ -0,0 +1,310 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, 2022 Red Hat, IBM Corporation and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Default docker image repos
+HPO_DOCKER_REPO="docker.io/kruize/hpo"
+
+# Default cluster
+CLUSTER_TYPE="native"
+PY_CMD="python3"
+LOGFILE="${PWD}/hpo.log"
+export N_TRIALS=3
+export N_JOBS=1
+
+function usage() {
+	echo "Usage: $0 [-s|-t] [-d] [-o hpo-image] [-r] [-c cluster-type]"
+	echo "s = start (default), t = terminate"
+	echo "r = restart hpo only"
+	echo "d = Don't start experiments"
+	echo "p = expose prometheus port"
```

Comment:
1. What are the valid cluster options ?
2. Is `d` option really used?
3. Why do we need prometheus port?

Reply:
Updated as required.